driver/src/examples/documentation/DocumentationSamples.java

```diff
@@ -603,6 +613,52 @@ public void testDeletions() {
         assertEquals(0, collection.count());
     }
 
+    @Test
+    public void testWatch() {
+        assumeTrue(isDiscoverableReplicaSet() && serverVersionAtLeast(3, 5));
+
+        final MongoCollection<Document> inventory = collection;
```

Comment:
In the other examples we just call it "collection" but here we call it "inventory" as in the DRIVERS ticket.  Is that what you prefer?